src/runtime_src/core/pcie/linux/CMakeLists.txt

```diff
@@ -52,11 +52,13 @@ target_link_libraries(xrt_core
   )
 
 install(TARGETS xrt_core
+  EXPORT xrt_core-targets
   LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} ${XRT_NAMELINK_SKIP}
   RUNTIME DESTINATION ${XRT_INSTALL_BIN_DIR}
 )
 
 install(TARGETS xrt_core
+  EXPORT xrt_core-targets
   ARCHIVE DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT}
   LIBRARY DESTINATION ${XRT_INSTALL_LIB_DIR} COMPONENT ${XRT_DEV_COMPONENT} ${XRT_NAMELINK_ONLY}
 )
```

Comment:
We also have emulation shim libraries that are released from runtime_src/core/pcie/emulation, don't these need be released  with the EXPORT option?

Reply:
I guess we should. I've never ran emulation so I didn't think of it. Let me see what I can do.